aggregations/db_tables/near_ecosystem_entities.py

```diff
@@ -40,4 +47,27 @@ def collect(self, requested_timestamp: int):
             read = csv.reader(csvFile, delimiter=",")
             result = list(read)
             csvFile.close()
+            csv_columns_count = len(result[0])
+
+            with self.analytics_connection.cursor() as analytics_cursor:
+                sql_columns = """
+                SELECT count(*) as columns 
+                FROM information_schema.columns 
+                WHERE table_name='near_ecosystem_entities'
+                """
+
+                analytics_cursor.execute(sql_columns)
+                sql_columns_count = analytics_cursor.fetchone()[0]
+                if sql_columns_count != csv_columns_count:
```

Comment:
What if the contents change in your next iteration, but the number of lines remains the same?

Reply:
This is to update the schema only.  yes, the rows still update.

Reply:
If so, please remove the csv file

Reply:
I simplified this to just drop and reload each day.  Complete overkill.